### DISCUSSION TITLE
Multi-VO requirements

### DISCUSSION THREAD

**Discussion by chrisburr** [DIRACGrid/diracx#19] [Extension requirements]
@atsareg mentioned that some services are limited to specific VOs and have different DBs for different VOs.

Please discuss any considerations and use cases you have so they can be considered.

**Comment by marianne013** [comment]
* Until we found https://github.com/DIRACGrid/DIRAC/issues/7075 we were convinced for 6 months that we would have to split the the FileCatalogDB for the different VOs. We were reluctant to do this, as there would be no way back, though we might still do this. Rucio would be the other obvious use case; we only have one or two contenders where rucio would make sense.
* At the moment the proxy database has a different user/password combination that the rest of the DBs for security reasons, though squalchemy might be good enough for this not to be necessary. 
* Tangentially related to this, it would probably be worth going through the transformation system with a fine tooth comb to make sure permissions to can be issued by VOs and possibly groups, i.e. everyone with the production role can see everything related to production jobs - while we have one or two groups that might be interested in the Transformation system we would be unlikely to roll it out for everyone. 
All a bit vague, I know.
